guides/common/modules/con_configuring-dhcp-integration.adoc

```diff
@@ -1,5 +1,5 @@
 [id="configuring-dhcp-integration"]
 = Configuring DHCP integration
 
-{Project} can manage IP leases on a DHCP server through a {SmartProxy}.
-This includes querying for available IP addresses, adding new reservations, and deleting existing reservations from the lease database.
+You can integrate DHCP with {Project} to automatically manage IP leases and boot configurations on a DHCP server during the provisioning of hosts.
+This helps to simplify the automated provisioning of new machines.
```

Comment:
```suggestion
This helps to simplify the automated provisioning of new hosts.
```

Suggestion: Do not use machine when refering to hosts.

context: I suggest that we stick to "host" in favor of machine/server etc.